imports/ui/templates/components/decision/title/title.js

```diff
@@ -118,17 +118,6 @@ Template.titleContent.helpers({
   declaration() {
     return getTitle(this);
   },
-  editable() {
```

Comment:
Not 100% we did not break anything on mobile here ; anyways this is going to be React soon.